config/reader.go

```diff
@@ -610,16 +610,6 @@ func processConfig(unprocessedConfig *Config, fromCloud bool, logger logging.Log
 		}
 	}
 
-	// pattern match configurations and resource log level configurations are both necessary for the logger registry
```

Comment:
All logic removed from this file has moved to `web/server/entrypoint.go`. As discussed offline, we do not want config processing to mutate the registry because a) we want fewer side-effects from these functions and b) we would have to pass the registry object down through these functions.